### PR TITLE
Repackage - Syslog (New AMA Dataconnector added)

### PR DESCRIPTION
**Pull request by v-rusraut**
Required items, please complete
   
   Change(s):
   - Repackage

   Reason for Change(s):
   - Addition of new Syslog AMA Data Connector

   Version Updated:
   - 3.0.0

   Testing Completed:
   - Done
  
image
image
image

### CODE REVIEW

File: Solutions/Syslog/Hunting Queries/CryptoThreatActivity.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 id: 005d6e3c-7453-46f6-aef3-f59695594545
 name: Suspicious crytocurrency mining related threat activity detected
 description: |
+    'This query detects terminated suspicious crypto mining payloads/processes. Attackers often halt other mining processes to gain more resources on compromised systems for their payload'
```
Comment (rahul0216) [inline Solutions/Syslog/Hunting Queries/CryptoThreatActivity.yaml:4]: Missing full stop.

File: Solutions/Syslog/Hunting Queries/SchedTaskAggregation.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 id: eb09da09-6f6c-4502-bf74-f7b9f1343539
 name: Linux scheduled task Aggregation
 description: |
+    'This query aggregates and charts cron job data based on unique user-command pairs. It shows the frequency of commands, the number of computers they’ve run on, and their percentage of total tenant computers'
```
Comment (rahul0216) [inline Solutions/Syslog/Hunting Queries/SchedTaskAggregation.yaml:4]: Missing full stop.

File: Solutions/Syslog/Hunting Queries/SchedTaskEditViaCrontab.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 id: 6f0f1821-5981-408a-930b-8b2ca60e9e6c
 name: Editing Linux scheduled tasks through Crontab
 description: |
+    'This query displays user edits to scheduled tasks via crontab, bucketed into 10-min intervals. It collects all actions by a user over seven days'
```
Comment (rahul0216) [inline Solutions/Syslog/Hunting Queries/SchedTaskEditViaCrontab.yaml:4]: Missing full stop.

File: Solutions/Syslog/Hunting Queries/squid_malformed_requests.yaml [hunk 4]
```diff
@@ -1,6 +1,8 @@
 id: edbeec9f-86b9-475d-8a42-cc7b95ad2baa
 name: Squid malformed requests
 description: |
+    'Malformed web requests can indicate network security devices. Hunting numerous requests from one source helps locate compromised hosts. Internal sites may need exclusion. Assumes default squid log format'
```
Comment (rahul0216) [inline Solutions/Syslog/Hunting Queries/squid_malformed_requests.yaml:4]: 'Malformed web requests are sometimes used for reconnaissance to detect the presence of network security devices. A large number of requests from a single source may indicate compromised hosts. Assumes default squid log format.'

File: Solutions/Syslog/Hunting Queries/squid_volume_anomalies.yaml [hunk 5]
```diff
@@ -1,6 +1,8 @@
 id: e472c490-4792-4f12-8b6b-6ab3e0404d35
 name: Squid data volume timeseries anomalies
 description: |
+    'Malware or data exfiltration can cause network data volume anomalies. This query detects such anomalies in bytes traversing a squid proxy, requiring further investigation. Assumes default squid log format'
```
Comment (rahul0216) [inline Solutions/Syslog/Hunting Queries/squid_volume_anomalies.yaml:4]: 'Malware or data exfiltration can cause network data volume anomalies. This query detects such anomalies in volume of bytes traversing a squid proxy. Anomalies require further investigation. Assumes default squid log format.'

**Review by rahul0216 — COMMENTED** [review]
Please see my comments inline.

File: Solutions/Syslog/Hunting Queries/CryptoThreatActivity.yaml (hunk 1 above)
Comment (v-rusraut) [reply Solutions/Syslog/Hunting Queries/CryptoThreatActivity.yaml]: Resolved

File: Solutions/Syslog/Hunting Queries/SchedTaskAggregation.yaml (hunk 2 above)
Comment (v-rusraut) [reply Solutions/Syslog/Hunting Queries/SchedTaskAggregation.yaml]: Resolved

File: Solutions/Syslog/Hunting Queries/SchedTaskEditViaCrontab.yaml (hunk 3 above)
Comment (v-rusraut) [reply Solutions/Syslog/Hunting Queries/SchedTaskEditViaCrontab.yaml]: Resolved

File: Solutions/Syslog/Hunting Queries/squid_malformed_requests.yaml (hunk 4 above)
Comment (v-rusraut) [reply Solutions/Syslog/Hunting Queries/squid_malformed_requests.yaml]: Resolved

File: Solutions/Syslog/Hunting Queries/squid_volume_anomalies.yaml (hunk 5 above)
Comment (v-rusraut) [reply Solutions/Syslog/Hunting Queries/squid_volume_anomalies.yaml]: Resolved

**Review by rahul0216 — DISMISSED** [review]
Description changes look good.

**Review by rahul0216 — DISMISSED** [review]
Description update look good.

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**